apps/sv/frontend/src/utils/buildAmuletConfigChanges.ts

```diff
@@ -11,39 +11,43 @@ import { ConfigChange } from './types';
 
 export function buildAmuletConfigChanges(
   before: Optional<AmuletConfig<'USD'>>,
-  after: Optional<AmuletConfig<'USD'>>
+  after: Optional<AmuletConfig<'USD'>>,
+  showAllFields: boolean = false
 ): ConfigChange[] {
   const changes = [
     {
-      fieldName: 'Tick Duration (microseconds)',
+      fieldName: 'tickDuration',
+      label: 'Tick Duration (microseconds)',
       currentValue: before?.tickDuration.microseconds || '',
       newValue: after?.tickDuration.microseconds || '',
     },
     {
-      fieldName: 'Transfer Preapproval Fee',
+      fieldName: 'transferPreapprovalFee',
```

Comment:
Saving the fieldName as we'll have to write an encoder to turn these into DsoConfig/AmuletConfig on submission.